apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -189,14 +199,25 @@ export default class LibraryPublisher extends React.Component {
     if (this.state.publishState === PublishState.ERROR_PUBLISH) {
       errorMessage = i18n.libraryPublishFail();
     }
+    if (this.state.publishState === PublishState.ERROR_UNPUBLISH) {
+      errorMessage = i18n.libraryUnPublishFail();
+    }
     return (
       <div>
         <p style={styles.alert}>{errorMessage}</p>
       </div>
     );
   };
 
+  unpublish = () => {
+    this.props.clientApi.delete(this.props.onUnpublishSuccess, error => {
+      console.warn(`Error publishing library: ${error}`);
```

Comment:
Do you want this console.warn long-term?

Reply:
Great question. I was thinking of this situation:
- teacher reaches out to us that students can't export their library
- we ask for more information including an image of the JS console

Was thinking the warning here would make the issue more obvious, but I'm open to doing this differently